packages/design-system/src/components/N8nActionDropdown/ActionDropdown.vue

```diff
@@ -92,6 +92,15 @@ export default Vue.extend({
 		onSelect(action: string) : void {
 			this.$emit('select', action);
 		},
+		onButtonBlur(event: FocusEvent): void {
+			const elementDropdown = this.$refs.elementDropdown;
+			// Hide dropdown when clicking outside of current document
+			if (elementDropdown && event.relatedTarget === null) {
+				// TODO: Disabling error here since this ref is always ElDropdown instance
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+				(elementDropdown as unknown as InstanceType<typeof ElDropdown>).hide();
```

Comment:
`as unknown` should not be needed.

Reply:
More deeply, `InstanceType<typeof ElDropdown>` is wrong as it is resolving to `any`, which is what the two lintings are flagging.